src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Certificates/StartCertificateHandlerTests/StartCertificateHandlerTests.cs

```diff
@@ -291,6 +301,8 @@ public void SetUp()
         {
             // Arrange
             learnerRecord.FundingModel = 81;
+            learnerRecord.GivenNames = learnerRecord.GivenNames.Substring(0, 1).ToUpper() + learnerRecord.GivenNames.Substring(1, learnerRecord.GivenNames.Length - 1);
```

Comment:
These two lines of code set the GivenNames and FamilyName values to mixed case, so that the name casing algorithm will not run. These are existing tests that don't need to be testing the algorithm at all. However, Moq tends to mock values that are either all uppercase or all lowercase which results in the algorithm being run and so the test then fails because the assertion compares the names before and after.

Reply:
The way that this is done in this branch is different to the way it is done in QF-602, where it is in a shared method to avoid repetition.